license-report.json

```diff
@@ -0,0 +1 @@
sler","relatedTo":"stuff","name":"ipfs-unixfs-importer","licensePeriod":"perpetual","material":"material","licenseType":"Apache-2.0 OR MIT","link":"git+https://github.com/ipfs/js-ipfs-unixfs.git","remoteVersion":"9.0.10","installedVersion":"9.0.10","definedVersion":"9.0.10","author":"n/a"},{"department":"kessler","relatedTo":"stuff","name":"level","licensePeriod":"perpetual","material":"material","licenseType":"MIT","link":"git+https://github.com/Level/level.git","remoteVersion":"8.0.0","installedVersion":"8.0.0","definedVersion":"8.0.0","author":"n/a"},{"department":"kessler","relatedTo":"stuff","name":"lodash","licensePeriod":"perpetual","material":"material","licenseType":"MIT","link":"git+https://github.com/lodash/lodash.git","remoteVersion":"4.17.21","installedVersion":"4.17.21","definedVersion":"4.17.21","author":"John-David Dalton <john.david.dalton@gmail.com>"},{"department":"kessler","relatedTo":"stuff","name":"lru-cache","licensePeriod":"perpetual","material":"material","licenseType":"ISC","link":"git://github.com/isaacs/node-lru-cache.git","remoteVersion":"7.12.0","installedVersion":"7.12.0","definedVersion":"7.12.0","author":"Isaac Z. Schlueter <i@izs.me>"},{"department":"kessler","relatedTo":"stuff","name":"ms","licensePeriod":"perpetual","material":"material","licenseType":"MIT","link":"git+https://github.com/vercel/ms.git","remoteVersion":"2.1.3","installedVersion":"2.1.3","definedVersion":"2.1.3","author":"n/a"},{"department":"kessler","relatedTo":"stuff","name":"multiformats","licensePeriod":"perpetual","material":"material","licenseType":"(Apache-2.0 AND MIT)","link":"git+https://github.com/multiformats/js-multiformats.git","remoteVersion":"9.6.4","installedVersion":"9.6.4","definedVersion":"9.6.4","author":"Mikeal Rogers <mikeal.rogers@gmail.com> (https://www.mikealrogers.com/)"},{"department":"kessler","relatedTo":"stuff","name":"randombytes","licensePeriod":"perpetual","material":"material","licenseType":"MIT","link":"git+ssh://git@github.com/crypto-browserify/randombytes.git","remoteVersion":"2.1.0","installedVersion":"2.1.0","definedVersion":"2.1.0","author":""},{"department":"kessler","relatedTo":"stuff","name":"search-index","licensePeriod":"perpetual","material":"material","licenseType":"MIT","link":"git+https://github.com/fergiemcdowall/search-index.git","remoteVersion":"3.1.3","installedVersion":"3.1.3","definedVersion":"3.1.3","author":"Fergus McDowall"},{"department":"kessler","relatedTo":"stuff","name":"uuid","licensePeriod":"perpetual","material":"material","licenseType":"MIT","link":"git+https://github.com/uuidjs/uuid.git","remoteVersion":"8.3.2","installedVersion":"8.3.2","definedVersion":"8.3.2","author":"n/a"},{"department":"kessler","relatedTo":"stuff","name":"varint","licensePeriod":"perpetual","material":"material","licenseType":"MIT","link":"git://github.com/chrisdickinson/varint.git","remoteVersion":"6.0.0","installedVersion":"6.0.0","definedVersion":"6.0.0","author":"Chris Dickinson <chris@neversaw.us>"}]
```

Comment:
Consider not checking in this and at the file to `.gitignore` instead.

Reply:
yeah oops, didn't mean to push that in, thanks.